relnotes/verify.feature.md

```diff
@@ -0,0 +1,9 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to [Sociomantic assert/enforce
+  policies](https://github.com/sociomantic-tsunami/sociomantic/blob/master/Code/assert_vs_enforce.md)
```

Comment:
Yup, awesome. I think that relying on me (one person) for reviewing structure of things even in that single repo is going to be too much of a bottleneck considering that we would want me to continue to focus on the overall strategy and setup of the program itself. But of course, if you like to discuss or need any feedback at all, I am super happy to comply, just poke me!